src/overlays/actors/ovl_En_Knight/z_en_knight.h

```diff
@@ -1,29 +1,136 @@
 #ifndef Z_EN_KNIGHT_H
 #define Z_EN_KNIGHT_H
 
+#include "assert.h"
 #include "global.h"
+#include "objects/object_knight/object_knight.h"
+
+static_assert((u32)IGOS_LIMB_MAX == (u32)KNIGHT_LIMB_MAX,
+              "The skeletons for Igos du Ikana and his aides must match in structure");
```

Comment:
Do these have to be the same, or would just taking the MAX of them both work?

Reply:
The entire skeleton has to be the same for two reasons
- many animations are shared between the skeletons
- some override/post limb draws are shared between the skeletons and assume that many limb indices are identical

Reply:
static assert pog